src/components/ConsoleBody/ConsoleBody.tsx

```diff
@@ -0,0 +1,84 @@
+import { ReactComponent as FormatIcon } from '@icons/format.svg';
+import React, { useEffect, useRef, useState, useCallback } from 'react';
```

Comment:
Please change all the imports in the app to that scheme: 1.) Fundamental imports like React etc. 2.)Fundamental libraries. 3.)Rest of the libraries 4.) Components 5.) Styles